documentation/List-of-Action-plugins.md

```diff
@@ -698,48 +729,6 @@ The "condition" field (common to all template_actions) can also override the ori
 
 ## Core Actions
 
-There are certain Actions that _must_ run on particular events to facilitate a standard application/review workflow process. We have called these **Core Actions**, and they have been collected in a single "core_mutations.js" file (for insertion into database via GraphQL). Each template (other than "User Registration") has this block slugged into it as a template literal (`${coreActions}`) in its Action insertion block, before any Actions that are specific to that template.
-
-Here is a summary of the core actions and the triggers that launch them:
-
-#### On Application Create:
-
-- Increment Stage (sets to Stage 1, also sets Status to "Draft")
-
-#### On Application Submit
-
-- Change Status (to "Submitted")
-- Trim Responses (removes Null responses and unchanged ones if re-submission)
-- Generate Review Assignments (for first level reviewers)
-- Update Reviews (statuses)
-- Cleanup Files
-
-#### On Application Restart (i.e. after "Changes Requested"):
-
-- Change Status (back to "Draft")
-
-#### On Review Self-Assign:
-
-- Update Review Assignment Status (for other reviewers)
-
-#### On Review Assign (by other)
-
-- Nothing yet (To-do?)
-
-#### On Review Create
-
-- Change Status (to "Draft")
-
-#### On Review Submit:
-
-- Change Status (to "Submitted")
-- Trim Responses
-- Update Review Statuses (for other reviews related to this review submission)
-- Increment Stage (if last level reviewer approves)
-- Generate Review Assignments (for next level review)
-- Update review response visibility (for applicant)
-- Change Status (Application, conditional on the review decision)
```

Comment:
It seemed wasteful to duplicate this information when it's already well-documented/commented in the actual code. (aka I don't want to maintain two lists!)